kinova_optimization/EEPoseGoals ee_pose_goals
---
kinova_optimization/JointAngles joint_angles
